designsafe/apps/rapid/tests.py

```diff
@@ -75,13 +75,13 @@ def test_get_opentopodata_center(self):
         self.assertContains(resp, "FeatureCollection")
 
     def test_get_opentopo_coordinates(self):
-        url = reverse('designsafe_rapid:get_opentopo_coordinates')
+        url = reverse('designsafe_rapid:get_opentopo_polygon_coordinates')
```

Comment:
we'll also need to update the tests by mocking the call via tapipy to get the file.  

after https://github.com/DesignSafe-CI/portal/pull/1373/files#r1688651134 is addressed, then the CI will fail on trying to get the file using tapis.   Now we can mock getting that file and return the example jsons we added previously to the repo.  

There could be different ways to mock the response from tapis.  We could mock TapiPy and the file.getContents method.  That might be the clearest way to do it. 🤔

Reply:
@nathanfranklin, for some clarification I am aware of mocking the files.getContents method to return the example json. 
But to mock Tapipy without auth details, can I use MagicMock or is there a preferred method

Reply:
addressed.